src/Searchbar.vue

```diff
@@ -49,6 +49,32 @@ export default {
       return value.replace(new RegExp(`(${phrase})`, 'gi'), '<mark>$1</mark>');
     },
   },
+  methods: {
+    down() {
+      if (this.current < this.items.length - 1) {
+        this.current += 1;
+        this.scrollWindow();
+      }
+    },
+    up() {
+      if (this.current > 0) {
+        this.current -= 1;
+        this.scrollWindow();
+      }
+    },
+    scrollWindow() {
```

Comment:
scrollListView() might be a bit more accurate. (I think the term "window" might be more suitable for the entire browser's window, rather than the component's window).